PalasoUIWindowsForms/ImageToolbox/ArtOfReadingChooser.cs

```diff
@@ -218,10 +227,98 @@ private void ArtOfReadingChooser_Load(object sender, EventArgs e)
 		   // _searchTermsBox.Text = @"flower";
 			_searchButton_Click(this,null);
 #endif
-
+			var searchControl = _searchTermsBox.TextBox.TheControl;
+			if (searchControl is TextBox)
+				searchControl.MinimumSize = _searchTermsBox.Size;
 			_messageLabel.BackColor = Color.White;
 		}
 
+		private void SetMessageLabelText()
+		{
+			var msg = "In the box above, type what you are searching for, then press ENTER.".Localize("ImageToolbox.EnterSearchTerms");
+			var searchLang = "English + Indonesian";
+			if (_searchLanguageMenu.Visible)
+				searchLang = _searchLanguageMenu.ToolTipText;
```

Comment:
The menu's Text contains only the native name of the language.  The tooltip contains the native name + the English name in parenthesis if it is in a nonRoman script or otherwise unlikely to be guessable by a literate European.